tools/sst_dump_tool.cc

```diff
@@ -14,6 +14,7 @@
 #include <inttypes.h>
 #include <iostream>
 #include <map>
+#include <memory>
```

Comment:
This is very weird. If I don't add this, shared_ptr<> used will be a wrong one which include a typeinfo call. Don't get why but it's a good habit to add it anyway, as the example: http://en.cppreference.com/w/cpp/memory/shared_ptr